packages/apps/docs/package.json

```diff
@@ -110,8 +110,7 @@
     "micromark-extension-frontmatter": "~1.1.0",
     "mini-css-extract-plugin": "2.7.6",
     "openapi-types": "~12.1.3",
-    "prettier": "~3.0.0",
-    "prettier-plugin-packagejson": "^2.4.5",
+    "prettier": "~3.0.3",
```

Comment:
if we're allowing prettier at root level dependency, then why are we keeping prettier again here?

Reply:
To keep them as much self-contained/isolated as possible.